### ISSUE TITLE
Block form no longer displaying

### ISSUE THREAD

**Issue by chrism** [thoughtbot/middleman-aria_current#16]
If I use the standard link_to helper in a block form ie

```erb
      <% link_to '/' do %>
        <h3>Home Link</h3>
      <% end %>
```

it displays fine, but when using the current_link_to helper the content within the helper is no longer displayed. ie

```erb
      <% current_link_to '/' do %>
        <h3>Home Link</h3>
      <% end %>
```

Is this intentional?

**Comment by tysongach** [comment]
@chrism Yeah, this is a known issue. I should documented that in the README.

We actually tried allowing blocks using the syntax you posted above, but it wasn’t working. Searching through old Middleman issues and on the forums, it appears that this has been a common issue.

I hope to take another crack at it soon, but for now, this block syntax should work:

```erb
<%= current_link_to("/with-block") { "Hello" } %>
```

**Comment by chrism** [comment]
Ok, great.

Thanks for letting me know I will try that.
